src/tests/routes/crates/versions/yank_unyank.rs

```diff
@@ -324,4 +326,26 @@ mod auth {
             json!({ "errors": [{ "detail": "must be logged in to perform that action" }] })
         );
     }
+
+    #[test]
+    fn admin() {
+        let (app, _, _) = prepare();
+
+        let admin = app.db_new_user("admin");
+
+        app.db(|conn| {
+            diesel::update(admin.as_model())
+                .set(users::is_admin.eq(true))
+                .execute(conn)
+                .unwrap();
+        });
+
+        let response = admin.yank(CRATE_NAME, CRATE_VERSION);
+        assert_eq!(response.status(), StatusCode::OK);
+        assert_eq!(response.into_json(), json!({ "ok": true }));
+
```

Comment:
Can we assert that the yank actually happened (and, below, that the unyank happened)?

Reply:
✅